pages/entropy/generate-random-numbers/evm.mdx

```diff
@@ -1,46 +1,68 @@
 Install the SDK using your package manager:
 
+<Tabs items={['hardhat', 'foundry']}>
+<Tabs.Tab>
+```shell copy
+npm install @pythnetwork/entropy-sdk-solidity
+```
+</Tabs.Tab>
+<Tabs.Tab>
 ```shell copy
+npm init -y
 npm install @pythnetwork/entropy-sdk-solidity
 ```
 
+Then add the following line to your `remappings.txt` :
+
+```text copy
+@pythnetwork/entropy-sdk-solidity/=node_modules/@pythnetwork/entropy-sdk-solidity
+```
+
+</Tabs.Tab>
+</Tabs>
+
 ## Setup
 
-The SDK exports a `IEntropyConsumer` interface that your contract should implement. The interface
-makes sure that your contract is compliant with the Entropy contract.
+The Solidity SDK exports two interfaces:
 
-The SDK also export `IEntropy` interface which you can use to interact with the Entropy contract.
-You will need the address of an Entropy contract on your blockchain.
-Consult the current [Entropy contract addresses](../contract-addresses) to find the address on your chain.
-Once you have a contract address, instantiate an `IEntropy` contract in your solidity contract:
+- [`IEntropyConsumer`](https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/ethereum/entropy_sdk/solidity/IEntropyConsumer.sol) - The interface that your contract should implement. It makes sure that your contract is compliant with the Entropy contract.
+- [`IEntropy`](https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/ethereum/entropy_sdk/solidity/IEntropy.sol) - The interface to interact with the Entropy contract.
+  You will need the address of an Entropy contract on your blockchain.
+  Consult the current [Entropy contract addresses](../contract-addresses) to find the address on your chain.
+  Once you have a contract address, instantiate an `IEntropy` contract in your solidity contract:
 
 ```solidity copy
 import { IEntropyConsumer } from "@pythnetwork/entropy-sdk-solidity/IEntropyConsumer.sol";
 import { IEntropy } from "@pythnetwork/entropy-sdk-solidity/IEntropy.sol";
 
-contract YourContract is IEntropyConsumer {
-  IEntropy entropy = IEntropy(<address>);
+// @param entropyAddress The address of the entropy contract.
+contract YourContract(address entropyAddress) is IEntropyConsumer {
+  IEntropy entropy = IEntropy(entropyAddress);
 }
-
 ```
 
-Entropy also requires selecting a randomness provider. The randomness provider is a third-party
+<Callout type="info">
+Entropy also requires selecting a **randomness provider**. The randomness provider is a third-party
 who participates in the generation process. Each provider is identified by an address and hosts
 a keeper service for fullfilling requests.
 
-The simplest way to choose a provider is to use the default provider.
+The simplest way to choose a provider is to use the default provider called [**Fortuna**](https://github.com/pyth-network/pyth-crosschain/tree/main/apps/fortuna).
```

Comment:
Fortuna is the name of the service or app we implemented. Anybody can run Fortuna, so I don't think it's right to call the default provider Fortuna.